src/components/Nav.js

```diff
@@ -1,85 +1,71 @@
-import React from "react";
+import React, { useState } from "react";
 
-export default class Nav extends React.Component {
-  state = {
-    navOpen: false
-  };
+export default function Nav() {
 
-  toggleSidenav() {
-    this.setState(prevState => ({
-      navOpen: !prevState.navOpen
-    }));
+  const [isOpen, setIsOpen] = useState(false);
+
+  const toggleSidenav = () => {
+    setIsOpen(!isOpen);
   }
-  render() {
-    return (
-      <nav
-        className={
-          "site-nav nav-padding" + (this.state.navOpen ? " expand-nav" : "")
-        }
-        id="site-nav"
-      >
-        <div className="site-nav-left" />
-        <div className="site-nav-middle">
-          <a
-            className="site-nav-logo"
-            href="https://coderadio.freecodecamp.org/"
+
+  const links = [
+    { href: "https://www.freecodecamp.org/news/", text: "News" },
+    { href: "https://www.freecodecamp.org/forum/", text: "Forum" },
+    { href: "https://www.freecodecamp.org/learn/", text: "Learn" }
```

Comment:
@ojeytonwilliams I thought we were moving away from hardcoding.  Shouldn't we use a config file of some sort?

Reply:
Generally speaking, yes, but this PR reduces duplication and overall makes the code easier to read, so I'm happy with it.

We'd want to pull things from a config file if we were creating multiple versions of the site with different links and text (if we translated Coderadio, say). At the moment it doesn't seem necessary.